db/migrate/20151104120951_remove_rate_from_chargeback_rate_detail.rb

```diff
@@ -0,0 +1,19 @@
+class RemoveRateFromChargebackRateDetail < ActiveRecord::Migration
+  class ChargebackRateDetail < ActiveRecord::Base
+    has_many :chargeback_tiers
+  end
+  class ChargebackTier < ActiveRecord::Base
+    belongs_to :chargeback_rate_detail
+  end
+
+  def change
+    ChargebackRateDetail.reset_column_information
+    ChargebackRateDetail.all.to_a.each do |detail|
+      if detail.respond_to?(:rate)
+        ChargebackTier.create(:chargeback_rate_detail_id => detail.id, :start => -Float::INFINITY, :end => 0, :fixed_rate => 0.0, :variable_rate => 0.0) # detail.rate)
+        ChargebackTier.create(:chargeback_rate_detail_id => detail.id, :start => 0, :end => Float::INFINITY, :fixed_rate => 0.0, :variable_rate => detail.rate)
+      end
+    end
+    remove_column :chargeback_rate_details, :rate, :string
```

Comment:
Can you do the schema change in a migration following this one?  It's best to keep schema changes separate from data migrations as it makes it easier to test and reason about.